Localization fixes

DrDudelsack opened the pull request:
Fixed localization for certain country names located in countries_l_english.yml

DrDudelsack committed:
Localization fixes

Fixed localization for country names located in countries_l_english.yml

Idhrendur commented on ImperatorToCk2/src/main/java/ImperatorToCK2/Importer.java:
```diff
@@ -665,8 +665,13 @@ else if (revoltName.equals("no")) {
                         }
+                        
+                        String adjName = tag.replace("_NAME","");
+                        String feuName = tag.replace("_FEUDATORY_NAME","_FEUDATORY_ADJECTIVE"); //for certain mission tags which have different formatting
```
Wouldn't ```feuAdj``` be closer to what you're doing? better yet, use the full words: ```feudatoryAdjective```

DrDudelsack replied:
I could use `feudatoryAdjective` instead, but `feuName` was quicker to type, lol, and my comment provides explanation on it's usage

Idhrendur replied:
It's easy to type, but is harder to understand when reviewing or maintaining. Abbreviations are a bad idea. See Effective Java Item 68 (Google it if you don't have a copy).

DrDudelsack committed:
Localization fixes 2

Fixed localization for revolts of a revolt of a revolt (and higher)

Idhrendur commented on ImperatorToCk2/src/main/java/ImperatorToCK2/Importer.java:
```diff
@@ -217,6 +217,22 @@ else if (qaaa.split("=")[0].equals( tab+tab+tab+"country_name" ) ) {
+                                                    if (qaaa.split("=")[0].equals("name")) {
+                                                        qaaa = qaaa.split("=")[1];
+                                                        qaaa = qaaa.substring(1,qaaa.length()-1);
+                                                        if (qaaa.equals("CIVILWAR_FACTION_NAME")) {
```
As Codacy note, this if statement is empty. Reverse the logic to make it simpler (unless this is a placeholder for logic in the if):

```
if (!qaaa.equals("CIVILWAR_FACTION_NAME")) {
  civilWarFlag = 1;
}
```

DrDudelsack replied:
The logic has been reversed. Is PR#37 now good to merge?

DrDudelsack committed:
Formatting change

DrDudelsack committed:
Character and Dynasty Localization Fixes

DrDudelsack committed:
Update Readme.txt

Updated the readme to include accurate information (Forum Release Thread will need to be updated upon release)

DrDudelsack committed:
Update Readme.txt

DrDudelsack committed:
Update Readme.txt
